### PR TITLE
Add sibeliaz recipe

### PR DESCRIPTION
**Pull request by iminkin**
Add whole genome aligner (SibeliaZ)[https://github.com/medvedevgroup/SibeliaZ].

### CODE REVIEW

**Comment by iminkin** [comment]
@bioconda-bot please add label

**Comment by iminkin** [comment]
@BiocondaBot please add label

File: recipes/sibeliaz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - openmp
+  host:
+    - twopaco
+    - spoa 4.0.3-0
+    - tbb-devel
+    - maf2synteny
```
Comment (dpryan79) [inline recipes/sibeliaz/meta.yaml:28]: You don't need any of these during run-time?

**Review by dpryan79 — CHANGES_REQUESTED** [review]
See comment

File: recipes/sibeliaz/meta.yaml (hunk 1 above)
Comment (iminkin) [reply recipes/sibeliaz/meta.yaml:29]: Oops, thanks for the catch!

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**